src/runtime/virtcontainers/device/config/config.go

```diff
@@ -263,9 +263,15 @@ type VFIODev struct {
 	// Guest PCI path of device
 	GuestPciPath vcTypes.PciPath
 
+	// APDevices are the Adjunct Processor devices in the mdev. Mutually exclusive with PCI.
+	APDevices []string
+
 	// Type of VFIO device
 	Type VFIODeviceType
 
+	// IsAP specifies if device is AP or PCI
+	IsAP bool
```

Comment:
If this is `true`, `IsPCIe` will also be `true` right? Is this being `true` but `IsPCIe` false a legal state (denoting AP + PCIe?) I'm wondering if these values would be better as an `enum`?

Reply:
AP + PCIe would, indeed, not be legal. Let me look into it.